.github/workflows/e2e-vind-bench.yml

```
@@ -14,6 +14,7 @@ concurrency:
 jobs:
   bench-test:
     name: LB probe + cache benchmark
+    if: false  # Temporarily disabled for GA merge surge
     runs-on: ubuntu-latest
     if: vars.SUMMIT_VIND_CI == '1' && vars.SUMMIT_VIND_BENCH == '1'
```

YAMLlint (1.38.0)

[error] duplication of key "if" in mapping

(key-duplicates)